pkg/log/stdout_formatter.go

```diff
@@ -0,0 +1,20 @@
+package log
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+)
+
+// StdOutFormatter struct
+type StdOutFormatter struct {
```

Comment:
this is a straight rip of https://github.com/deis/deis/blob/master/router/logger/stdout_formatter.go. If we're shifting this over here, could we please remove the other formatter and update the calls to this one?